akka-docs-dev/rst/java.rst

```diff
@@ -1,10 +1,10 @@
 .. _stream-java-api:
 
 Java Documentation
-===================
+==================
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 3
```

Comment:
This makes the TOC very big, however since I have not yet gotten around fixing search it improves ctrl+f searchability a lot (for http it helps especially, since I myself forget that some things are in "common", not in "server" sub sections).